### ISSUE TITLE
Error decoding payload with accents

### ISSUE THREAD

**Issue by ericx0099** [RobDWaller/ReallySimpleJWT#80]
If any property of the payload is a string with accents this function ( Token::validate($token, $secret);) returns false without entering in any catch clause.
Example:  "name": "Gerard Hernàndez",

**Comment by proyectosticsanjuan** [comment]
> If any property of the payload is a string with accents this function ( Token::validate($token, $secret);) returns false without entering in any catch clause. Example: "name": "Gerard Hernàndez",

Hi Eric. Could you solve it in any way?

**Comment by kabeza** [comment]
> If any property of the payload is a string with accents this function ( Token::validate($token, $secret);) returns false without entering in any catch clause. Example: "name": "Gerard Hernàndez",

Hi Eric. Could you solve it in any way?

**Comment by ericx0099** [comment]
> > If any property of the payload is a string with accents this function ( Token::validate($token, $secret);) returns false without entering in any catch clause. Example: "name": "Gerard Hernàndez",
> 
> Hi Eric. Could you solve it in any way?

Yes, i encoded with base64 the full payload, and the result of the base64 is what i encrypt/decrypt with jwt.

**Comment by kabeza** [comment]
> > > If any property of the payload is a string with accents this function ( Token::validate($token, $secret);) returns false without entering in any catch clause. Example: "name": "Gerard Hernàndez",
> > 
> > 
> > Hi Eric. Could you solve it in any way?
> 
> Yes, i encoded with base64 the full payload, and the result of the base64 is what i encrypt/decrypt with jwt.

Thanks for the tip
